[2024-03-18T08:41:01.118+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sftp_sync.sync_files_task manual__2024-03-18T08:40:52.610941+00:00 [queued]>
[2024-03-18T08:41:01.134+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sftp_sync.sync_files_task manual__2024-03-18T08:40:52.610941+00:00 [queued]>
[2024-03-18T08:41:01.134+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-18T08:41:01.161+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): sync_files_task> on 2024-03-18 08:40:52.610941+00:00
[2024-03-18T08:41:01.172+0000] {standard_task_runner.py:60} INFO - Started process 1941 to run task
[2024-03-18T08:41:01.185+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sftp_sync', 'sync_files_task', 'manual__2024-03-18T08:40:52.610941+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/sftp_sync_between_source_and_target.py', '--cfg-path', '/tmp/tmpgo4n17dg']
[2024-03-18T08:41:01.191+0000] {standard_task_runner.py:88} INFO - Job 88: Subtask sync_files_task
[2024-03-18T08:41:01.372+0000] {task_command.py:423} INFO - Running <TaskInstance: sftp_sync.sync_files_task manual__2024-03-18T08:40:52.610941+00:00 [running]> on host fa05d1b87173
[2024-03-18T08:41:01.577+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sftp_sync' AIRFLOW_CTX_TASK_ID='sync_files_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-18T08:40:52.610941+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-18T08:40:52.610941+00:00'
[2024-03-18T08:41:01.594+0000] {ssh.py:301} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2024-03-18T08:41:01.630+0000] {transport.py:1909} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
[2024-03-18T08:41:02.007+0000] {transport.py:1909} INFO - Authentication (password) successful!
[2024-03-18T08:41:02.129+0000] {sftp.py:169} INFO - [chan 0] Opened sftp connection (server version 3)
[2024-03-18T08:41:02.144+0000] {ssh.py:301} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2024-03-18T08:41:02.230+0000] {transport.py:1909} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
[2024-03-18T08:41:02.594+0000] {transport.py:1909} INFO - Authentication (password) successful!
[2024-03-18T08:41:02.689+0000] {sftp.py:169} INFO - [chan 0] Opened sftp connection (server version 3)
[2024-03-18T08:41:02.701+0000] {python.py:202} INFO - Done. Returned value was: Synced 0 new files.
[2024-03-18T08:41:02.881+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=sftp_sync, task_id=sync_files_task, execution_date=20240318T084052, start_date=20240318T084101, end_date=20240318T084102
[2024-03-18T08:41:03.065+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-18T08:41:03.181+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
